### ISSUE TITLE
watches files inside ./node_modules

### ISSUE THREAD

**Issue by andykais** [johnpapa/lite-server#55]
starting the [angular2 quickstart](https://github.com/angular/quickstart) server `npm run lite-server` watches for changes in `node_modules` causing it to reload over and over again. Adding a `my-bs-config.json` with this inside
```
{
  "files": [
    "./app/**/*.{html,htm,css,js}",
    "./*.{html,css}"
  ]
}
```
appears to stop it from constantly reloading itself but it still is watching files withing `node_modules`
```
[1] [BS] Serving files from: ./
[1] [BS] Watching files...
[1] 16.03.17 13:10:24 304 GET /index.html
[1] 16.03.17 13:10:24 304 GET /styles.css
[1] 16.03.17 13:10:24 304 GET /node_modules/es6-shim/es6-shim.min.js
[1] 16.03.17 13:10:24 304 GET /node_modules/systemjs/dist/system-polyfills.js
[1] 16.03.17 13:10:24 304 GET /node_modules/angular2/es6/dev/src/testing/shims_for_IE.js
[1] 16.03.17 13:10:24 304 GET /node_modules/angular2/bundles/angular2-polyfills.js
[1] 16.03.17 13:10:24 304 GET /node_modules/systemjs/dist/system.src.js
[1] 16.03.17 13:10:24 304 GET /node_modules/rxjs/bundles/Rx.js
[1] 16.03.17 13:10:24 304 GET /node_modules/angular2/bundles/angular2.dev.js
[1] 16.03.17 13:10:24 304 GET /node_modules/angular2/bundles/angular2-polyfills.js
[1] 16.03.17 13:10:24 304 GET /node_modules/systemjs/dist/system.src.js
[1] 16.03.17 13:10:24 304 GET /node_modules/rxjs/bundles/Rx.js
[1] 16.03.17 13:10:24 304 GET /node_modules/angular2/bundles/angular2.dev.js
[1] 16.03.17 13:10:24 304 GET /app/main.js
[1] 16.03.17 13:10:24 304 GET /app/app.component.js
```

I am running: 
```
linux mint rafaela
node v5.6.0
npm v3.6.0
```

**Comment by cgmartin** [comment]
A workaround, until #57 is published, is to add the following to your `bs_config.json` file:
```
{
    "watchOptions": {
        "ignored": "node_modules"
    }
}
```

**Comment by johnpapa** [comment]
published in 2.2.0

#closed
